index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Nabil's Blog</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="main.css">
+    </head>
+    <body>
+        <div class="chom">
+            <div class="chom-triangle"></div>
+            <div class="chom-rect"></div>
+            <div class="chom-top"></div>
+        </div>
+        <div class="head-triangle">
+            <div class="left-corner"></div>
+            <div class="right-corner"></div>
+            <div class="white-triangle"></div>
+            <div class="attic">
+                <div class="base">
+                    <div class="door">
+                        <div class="handle"></div>
+                    </div>
+                    <table class="left-window">
+                        <tr><td></td><td></td></tr>
```

Comment:
Using `::before` and `::after` elements with borders (`black 1px solid`) can also help you to solve this problem, so you can get rid of the table code.